src/Assembler.hpp

```diff
@@ -2035,6 +2035,21 @@ class shasta::Assembler :
     void exploreMarkerGraphEdge(const vector<string>&, ostream&);
     void exploreMarkerCoverage(const vector<string>&, ostream&);
     void exploreMarkerGraphInducedAlignment(const vector<string>&, ostream&);
+    void renderEditableAlignmentConfig(
+        const int method,
+        const uint64_t maxSkip,
+        const uint64_t maxDrift,
+        const uint32_t maxMarkerFrequency,
+        const uint64_t minAlignedMarkerCount,
+        const double minAlignedFraction,
+        const uint64_t maxTrim,
+        const int matchScore,
+        const int mismatchScore,
+        const int gapScore,
+        const double downsamplingFactor,
```

Comment:
I was wondering if I could reuse `AssemblerOptions::AlignOptions` here, but turns out that I can't really forward declare an inner class :-)

I wondered if I could reuse `ComputeAllAlignmentsData` here, but decided against it in order to keep the compute and render code separate.

So I ended up with this method signature.

Reply:
If you found it useful, it would be ok to `#include "AssemblerOptions.hpp"` in `Assembler.hpp`. Then you would be able to use `AlignOptions` there. Or conceivably even make this function a member of `AlignOptions`. Alternatively, moving `AlignOptions` out of `AssemblerOptions` would be fine too. Or you can leave it as is, perfectly fine.

I agree with not using `ComputeAllAlignmentsData`. I prefer those classes used to communicate with threads to be used by one function each. That makes it easier to make changes, when needed, without having to test other pieces of functionality.